governance/votes/Executive vote - July 11, 2023.md

```diff
@@ -0,0 +1,122 @@
+- Reduce vow.bump by 25,000 DAI from 30,000 DAI to 5,000 DAI
+- Set hop to 1,577 seconds
+- Set want to 0.98
+- Set receiver to MCD_PAUSE_PROXY
+- Set pip to TBD
+
+### Stability Scope Parameter Changes
+
+As per the Stability Scope Facilitator, SES, [the following changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238) will be implemented. 
+
+#### [Dai Savings Rate(DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease
+
+- Reduce DSR by 0.30% from 3.49% to 3.19%
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Decreases
+
+- Reduce the ETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the ETH-B Stability Fee (SF) by 0.30% from 4.24% to 3.94%
+- Reduce the ETH-C Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) Decreases
+
+- Reduce WSTETH-A Liquidation Ratio by 10% from 160% to 150%
+- Reduce WSTETH-B Liquidation Ratio by 10% from 185% to 175%
+- Reduce RETH-A Liquidation Ratio by 20% from 170% to 150%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+-  Set CRVV1ETHSTETH-A [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) to 0
+-  Remove CRVV1ETHSTETH-A from autoline
+
+### Aligned Delegate Compensation Distribution for June 2023
+
+TBD
+
+### Budget Streams and Payments
+
+As per [this poll](https://forum.makerdao.com/t/mip102c2-sp8-mip-amendment-subproposals/20761/8), the following streams will be set up for Chronicle Labs
+
+| Budget         | DAI Amount | MKR Amount | Start Date | End Date   | Destination Address                        |
+| -------------- | ---------- | ---------- | ---------- | ---------- | ------------------------------------------ |
+| Chronicle Labs | 3,721,800  | 2,216.4    | 2023-07-01 | 2024-06-30 | 0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f |
+
+As per [their successful MKR Vesting Proposal](https://forum.makerdao.com/t/mip40c3-sp36-deco-fixed-rate-core-unit-mkr-budget/10226), **125 MKR** will be transferred to the DECO wallet 0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f, if this executive proposal passes.
+
+### DssCron Housekeeping Additions
+
+As per [the SES Facilitator](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292), the following active jobs will be added to the next spell, which will bring information on contracts authorized on the Pause Proxy to the Chainlog.
```

Comment:
I'm aware you pulled this from the forum post, but the forum post is not a great description of what is happening here and why. If you are uncertain how something works, or why it's happening, go and talk to the devs and find out. Alternatively, follow the link to where sunbreak explains it, and use that. Compare to the official version. 

What is actually happening here is that existing keeper job addresses are being added to the chainlog.

Reply:
Also, with this hat, SES are the Protocol Scope Facilitators.